presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -1337,6 +1337,11 @@ public void testSelectFieldsWithCapitalLetters()
         }
     }
 
+    @Test
+    public void testTaskTable() {
```

Comment:
Is it guaranteed to have at least 2 tasks? Just ensuring the test will be stable.
Can we add more tests corresponding to the `select * from system.runtime.tasks` tasks?

Reply:
Added comment.